db/index.js

```diff
@@ -1,13 +1,87 @@
 // Connect to DB
 const { Client } = require('pg');
-const DB_NAME = 'change-this-name'
-const DB_URL = process.env.DATABASE_URL || `postgres://${ DB_NAME }`;
+// I chose vinyldb as our DB name
+const DB_NAME = 'vinyldb'
+const DB_URL = process.env.DATABASE_URL || `postgres://localhost:5432/${ DB_NAME }`;
 const client = new Client(DB_URL);
 
 // database methods
+async function createAlbums({
+  album_name,
+  artist,
+  year,
+  price,
+  quantity,
+  reorder,
+  img_url
+}) {
+  try {
+    const { rows: [ album ] } = await client.query(`
+      INSERT INTO products(album_name, artist, year, price, quantity, reorder_number, img_url)
+      VALUES($1, $2, $3, $4, $5, $6, $7)
+      ON CONFLICT (album_name) DO NOTHING
+      RETURNING *;
+    `, [album_name, artist, year, price, quantity, reorder, img_url]);
+
+    return album;
+  } catch (error) {
+    throw error;
+  }
+}
+
+async function createGenres({
+  genre,
+}) {
+  try {
+    const { rows: [ genrename ] } = await client.query(`
```

Comment:
Nitpick I know but maybe camel case genreName?

Reply:
I refrained myself on this lolo

Reply:
I guess. I was using username which is all lower case but ill change it